salt/config/__init__.py

```diff
@@ -1958,6 +1953,30 @@ def _validate_file_roots(file_roots):
     return file_roots
 
 
+def _validate_pillar_roots(pillar_roots):
```

Comment:
Does something magically call this function? I'm not seeing a change to call `_validate_pillar_roots`, so just mildly confused 😛

Reply:
@waynew Good catch, fixed.

Reply:
Glad I'm not crazy 😀